docs/index.rst

```diff
@@ -6,54 +6,76 @@ provides a simple interface for performing aperture photometry of either a singl
 image or a directory with multiple images. It is designed to be easy to use for both
 non-programmers and programmers.
 
-
-Getting Started
-===============
-
 Installation
-------------
+============
+
 
 Testing
-+++++++
+-------
 
 If you are testing a pre-release version of stellarphot we recommend setting up
 a virtual environment and installing stellarphot in this environment.
 
-Creating an environment with `conda` or `mamba`:
+Only use one of the methods below for making a virtual environment.
 
-```bash
-mamba create -n stellarphot-test python=3.11
-mamba activate stellarphot-test
-pip install --pre --upgrade stellarphot
-```
+Creating an environment with `conda` or `mamba`::
 
-Creating an environment with `virtualenv`:
+    mamba create -n stellarphot-test python=3.11
+    mamba activate stellarphot-test
```

Comment:
For what it is worth, `mamba activate stellarphot-test` gave me an error that said I should use `mamba init` but then trying to use `mamba init` failed.  I ended up using `conda activate stellarphot-test` just fine.  If you have a current installation of `conda` it should automatically use the `mamba` backend eliminating the need to explicitly call `mamba`.

Reply:
I use the miniforge prompt for doing all of this and have not run into any issues with the commands beyond the one mentioned farther down this review. But there may need to be more explanation on the commands and how they might need to be modified for differnet launchers.

Reply:
Tried to address this a little bit...

Reply:
OK.